src/components/comments/types.ts

```diff
@@ -8,6 +8,14 @@ export type Annotation = {
   comments: Comment[];
 };
 
+export type Participant = {
+  id: string;
+  name: string;
+  avatar: string;
+  email: string;
+  position?: string;
+};
```

Comment:
1. Rename the ParticipantApi to ParticipantByCommentsApi.
2. Create the new participant by group in the participant.types.ts

Like this: 

```ts

export type ParticipantByCommentsApi = {
  uuid: string;
  participantId: string;
  name: string;
  createdAt: string;
  avatar: string;
};


export type ParticipantByGroupApi = {
  id: string;
  name: string;
  avatar: string;
  email: string;
  position?: string;
};

```


And it's better if you declare another type with position, because it's not coming through the API, it's only used in one place. Something like:  

```ts

export type ParticipantByGrouWithPosition = ParticipantByGroupApi & {
  position: string;
}


```